app/serializers/api/admin/line_item_serializer.rb

```diff
@@ -13,7 +13,7 @@ def units_variant
     Api::Admin::UnitsVariantSerializer.new(object.variant).serializable_hash
   end
 
-  def unit_value
-    object.unit_value.to_f
+  def final_weight_volume
+    object.final_weight_volume.to_f
```

Comment:
Yeah not sure. The serializer I believe was changed by @oeoeaio in some bulk management refactoring? So I just kept the cast to float as per his changes.